go/pkg/command/command.go

```diff
@@ -71,6 +71,17 @@ func (s SymlinkBehaviorType) String() string {
 	return fmt.Sprintf("InvalidSymlinkBehaviorType(%d)", s)
 }
 
+func SymlinkBehavior(s string) SymlinkBehaviorType {
+  switch s {
+  case "resolve":
```

Comment:
nit: allow the values specified in `symlinkBehaviorType` as well.

Reply:
Will do. Is it fine to leave the shorter strings as acceptable too or only allow the full names?